### DISCUSSION TITLE
extensions didn't deployed

### DISCUSSION THREAD

**Discussion by nawwaf15** [AntlerVC/firetable#1345] [Support / Q&A] [answered]
I am still a beginner at this, I started with the demo project for Whatsapp general AI. I followed every step but when I tried to send a message to the number the AI didn't reply. I checked everything then I found that the extensions field is empty Although I deployed it. how to fix this issue? and how to make the extensions get filled like the introduction video?
![Screen Shot 2023-07-17 at 1 43 50 PM](https://github.com/rowyio/rowy/assets/139074132/04aca99f-d3ec-42f3-830a-a8d0f0963df8)

**Comment by BeeBombshell** [comment, marked as answer]
Hey @nawwaf15, are you using the WhatsApp ChatGPT template? For that, the request to OpenAI is sent via the Derivative field. (See screenshot attached)
<img width="1800" alt="image" src="https://github.com/rowyio/rowy/assets/53828745/aceda7c5-8ffd-4b38-91d4-c70a9d05a44f">

One thing to make sure, do check if you have correctly set up your secrets in the GCP Secret Manager as prompted in the setup process. (For Example: OpenAI API has be set with the name `openai`)
